lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/ProcessJourneyEventHandler.java

```diff
@@ -245,4 +265,46 @@ private Map<IpvJourneyTypes, StateMachine> loadStateMachines(
         }
         return stateMachinesMap;
     }
+
+    @Tracing
+    private void checkIfStartingMitigationJourney(
```

Comment:
I still don't love this name - can we call it something more like `sendMitigationStartAuditEvent` (or if you insist, `sendMitigationStartAuditEventIfRequired`), so it's clear what it does?

(The fact it sends the audit event is the important bit - the fact it checks the journey step property is an implementation detail!)